test/integration/create-next-app/package-manager/pnpm.test.ts

```diff
@@ -12,22 +10,28 @@ import {
 const lockFile = 'pnpm-lock.yaml'
 const files = [...DEFAULT_FILES, lockFile]
 
-let nextInstall: Awaited<ReturnType<typeof createNextInstall>>
-beforeAll(async () => {
-  nextInstall = await createNextInstall({
-    parentSpan: trace('test'),
-    keepRepoDir: Boolean(process.env.NEXT_TEST_SKIP_CLEANUP),
-  })
-})
+describe('create-next-app with package manager pnpm', () => {
+  let nextTgzFilename: string
 
-beforeEach(async () => {
-  await command('pnpm', ['--version'])
-    // install pnpm if not available
-    .catch(() => command('corepack', ['prepare', '--activate', 'pnpm@latest']))
-    .catch(() => command('npm', ['i', '-g', 'pnpm']))
-})
+  beforeAll(async () => {
+    if (!process.env.NEXT_TEST_PKG_PATHS) {
+      throw new Error('This test needs to be run with `node run-tests.js`.')
+    }
+
+    const pkgPaths = new Map<string, string>(
+      JSON.parse(process.env.NEXT_TEST_PKG_PATHS)
+    )
+
+    nextTgzFilename = pkgPaths.get('next')
+
+    await command('pnpm', ['--version'])
+      // install pnpm if not available
```

Comment:
When would this not be available?

Reply:
I don't know, I just moved this line here. Maybe this is from the days when Next.js was still using yarn? Should be safe to remove I guess.

Reply:
Right, this was here before. Fine to me to leave as-is.